lib/workers/repository/dashboard-package-files.ts

```diff
@@ -0,0 +1,60 @@
+import type { RenovateConfig } from '../../config/types';
+import type { PackageFile } from '../../modules/manager/types';
+
+export class DashboardPackageFiles {
```

Comment:
```suggestion
export class PackageFiles {
```

I think best to make this class and filename more generic - `package-files.ts`. We might use it for more than dashboard later.